deps/python-kombu/python-kombu.spec

```diff
@@ -11,7 +11,7 @@ Name:           python-%{srcname}
 # The Fedora package is using epoch 1, so we need to also do that to make sure ours gets installed
 Epoch:          1
 Version:        3.0.24
-Release:        5.pulp%{?dist}
+Release:        7.pulp%{?dist}
```

Comment:
why does this go from 5 to 7? There isn't a shortage of integers, just curious :koala: